pkg/agent/flowexporter/connections/connections.go

```diff
@@ -174,6 +174,10 @@ func (cs *connectionStore) addOrUpdateConn(conn *flowexporter.Connection) {
 		// IDs stored in the connection label.
 		if len(conn.Labels) != 0 {
 			klog.V(4).Infof("connection label: %x; label masks: %x", conn.Labels, conn.LabelsMask)
+			// We always expect labels from conntrack dumper to be added in little-endian format right now
+			// In kernel datapath, the labels uses the "native" endianness for the system, which are little-endian
+			// on most of the modern CPUs based on x86 architecture like Intel, AMD, etc.
+			// TODO: detect the endianness of the system when initializing conntrack dumper to handle situations on big-endian platforms.
```

Comment:
The comment and the context look good and clear. The TODO part should be in conntrack_linux.go, where this has to be fixed IMO. 
Sorry, maybe I was not clear in my last comment--I meant to add those extra comments here but keep the TODO there.

Reply:
Thanks, addressed.